wiki/Tournaments/Official_support/en.md

```diff
@@ -86,7 +92,17 @@ Once the tournament has concluded, the tournament organisers will need to submit
 
 Tournament organisers are expected to ensure that their tournaments run smoothly and with minimal disruption where possible.
 
-Users under an active tournament ban are expected to disclose their tournament ban status to any officially supported tournament they intend to help with. They may be enlisted as [helpers](#what-is-considered-staff) at the host's discretion, but we encourage careful consideration of such choices as said individuals have already infringed upon the rules once (or more). They may not act as [staff](#what-is-considered-staff) UNLESS the host requests an exemption for them when sending in their initial request for support. The account support team will then assess these on a case-by-case basis with the user's history in mind.
+Users under an active tournament ban are expected to disclose their tournament ban status to any officially supported tournament they intend to help with.
+
+With regards to staff roles, the following rules apply:
+
+- Streamers, commentators, and graphic designers may play in the tournament.
+  - Users under an active tournament ban may enlist as one of these roles, but we encourage careful consideration of such choices as said individuals have already infringed upon the rules once (or more).
```

Comment:
what

> - Streamers, commentators, and graphic designers may play in the tournament.
>   - Users under an active tournament ban may enlist as [streamers, commentators, or graphic designers]

Reply:
i'm referring to the implication that tournament-banned users may enlist under these roles and therefore play in the tournament

Reply:
We can move both this and the other indented block to their own paragraph.